src/common/buffer.cc

```diff
@@ -2155,3 +2136,79 @@ MEMPOOL_DEFINE_OBJECT_FACTORY(buffer::raw_unshareable, buffer_raw_unshareable,
 MEMPOOL_DEFINE_OBJECT_FACTORY(buffer::raw_static, buffer_raw_static,
 			      buffer_meta);
 
+
+namespace ceph::buffer {
+
+class buffer_error_category : public ceph::converting_category {
+public:
+  buffer_error_category(){}
+  const char* name() const noexcept override;
+  std::string message(int ev) const noexcept override;
```

Comment:
You're correct. I knew some methods on error_category had to be noexcept, but this isn't one of them.

(In fact some people were wanting to return a string_view precisely so it could be.)

Reply:
Fixed!